.github/workflows/Build.yml

```diff
@@ -94,6 +94,10 @@ jobs:
       uses: r-lib/actions/setup-r@v2
       with:
         r-version: 'latest'
+    - name: Install Examples R package dependencies
+      if: contains(matrix.ctest-cache, 'WRAP_R')
+      run: |
```

Comment:
This change set does not contain any R code changes. Is the a missing dependency with the R example?